packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+
+export default {
+  title: 'Emails/Cad Reminder',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
+
+export const DesktopDevice = Template.bind({});
+DesktopDevice.args = {
+  template: 'cadReminder',
+  description:
+    'The Connect Another Device Reminder is sent when [TODO: documentation].',
```

Comment:
@LZoog Do have we have this `description` documented somewhere?

Reply:
We don't, which is exactly why having some documentation here will be great. This isn't something I think you should or can fill out though since digging around for the context of each one would take time away from converting them over, so instead, we can file a follow up ticket for someone on the FxA team to help out with a bit later (unless there's some time left after template conversions).

Though, I'm kind of thinking to make the purpose of this more obvious, we could rename it from `description` to `doc`. `doc` is a property [we use to document config setting descriptions](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/config/index.ts#L27) with a package called convict so it'd lead to a tiny bit of consistency/clarity. Maybe when you work on finishing the `cadReminder` conversion in #9255 we can at least edit this one to show what it'll look like to have one in place as an example, leave the rest as blank to do later.